archetypes.js

```diff
@@ -0,0 +1,1484 @@
+const emptyBuild = {
+  position: 'None',
+  shortPosition: 'None',
+  archetype: 'None',
+  baseTpe: 0,
+  strengthMin: 0,
+  strengthMax: 0,
+  agilityMin: 0,
+  agilityMax: 0,
+  armMin: 0,
+  armMax: 0,
+  intelligenceMin: 0,
+  intelligenceMax: 0,
+  throwingAccuracyMin: 0,
+  throwingAccuracyMax: 0,
+  tacklingMin: 0,
+  tacklingMax: 0,
+  speedMin: 0,
+  speedMax: 0,
+  handsMin: 0,
+  handsMax: 0,
+  passBlockingMin: 0,
+  passBlockingMax: 0,
+  runBlockingMin: 0,
+  runBlockingMax: 0,
+  enduranceMin: 0,
+  enduranceMax: 0,
+  kickPowerMin: 0,
+  kickPowerMax: 0,
+  kickAccuracyMin: 0,
+  kickaccuracyMax: 0
+};
+
+const safetyCenterFielder = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Center Fielder',
+  baseTpe: 466,
+  strengthMin: 40,
+  strengthMax: 70,
+  agilityMin: 70,
+  agilityMax: 95,
+  armMin: 1,
+  armMax: 25,
+  intelligenceMin: 40,
+  intelligenceMax: 80,
+  throwingAccuracyMin: 1,
+  throwingAccuracyMax: 25,
+  tacklingMin: 50,
+  tacklingMax: 75,
+  speedMin: 70,
+  speedMax: 100,
+  handsMin: 55,
+  handsMax: 75,
+  passBlockingMin: 1,
+  passBlockingMax: 25,
+  runBlockingMin: 1,
+  runBlockingMax: 25,
+  enduranceMin: 70,
+  enduranceMax: 100,
+  kickPowerMin: 1,
+  kickPowerMax: 25,
+  kickAccuracyMin: 1,
+  kickaccuracyMax: 25
+};
+
+const safetyPlayermaker = {
+  position: 'Safety',
+  shortPosition: 'S',
+  archetype: 'Playmaker',
+  baseTpe: 456,
```

Comment:
template: 401 tpe + 50 available, shouldn't it be 451?

Reply:
(MAX: 70) Strength: 40 (40 TPE)
(MAX: 95) Agility: 70 (90 TPE)
(MAX: 25) Arm: 1 (1 TPE)
(MAX: 80) Intelligence: 50 (50 TPE)
(MAX: 25) Throwing Accuracy: 1 (1 TPE)
(MAX: 75) Tackling: 50 (50 TPE)
(MAX: 95) Speed: 65 (80 TPE)
(MAX: 85) Hands: 55 (60 TPE)
(MAX: 25) Pass Blocking: 1 (1 TPE)
(MAX: 25) Run Blocking: 1 (1 TPE)
(MAX: 100) Endurance: 65 (80 TPE)
(MAX: 25) Kick Power: 1 (1 TPE)
(MAX: 25) Kick Accuracy: 1  (1 TPE)

40+90+1+50+1+50+80+60+1+1+80+1+1 = 456